tracer/src/Datadog.Trace/AgentProcessManager.cs

```diff
@@ -56,29 +52,56 @@ internal enum ProcessState
             Healthy
         }
 
+        /// <summary>
+        /// Invoked by the loader
+        /// </summary>
         public static void Initialize()
         {
             try
             {
-                if (string.IsNullOrWhiteSpace(TraceAgentMetadata.ProcessPath))
-                {
-                    return;
-                }
-
                 if (DomainMetadata.Instance.ShouldAvoidAppDomain())
                 {
                     Log.Information("Skipping process manager initialization for AppDomain: {AppDomain}", DomainMetadata.Instance.AppDomainName);
                     return;
                 }
 
-                if (!Directory.Exists(TraceAgentMetadata.DirectoryPath))
+                var customTracingEnabled = EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.AasEnableCustomTracing, false);
+                var needsDogStatsD = EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.AasEnableCustomMetrics, false);
+                var automaticTraceEnabled = EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.TraceEnabled, true);
+
+                EnvironmentHelpers.GetBooleanEnvironmentVariable(ConfigurationKeys.AasEnableCustomTracing, false);
+
+                if ((customTracingEnabled || automaticTraceEnabled) && string.IsNullOrWhiteSpace(TraceAgentMetadata.ProcessPath))
+                {
+                    Log.Warning("Requested to start the Trace Agent but the process path hasn't been supplied in environment.");
+                }
+                else if (!Directory.Exists(TraceAgentMetadata.DirectoryPath))
                 {
                     Log.Warning("Directory for trace agent does not exist: {Directory}", TraceAgentMetadata.DirectoryPath);
-                    return;
```

Comment:
We'll fall into this branch if `customTracingEnabled==false && automaticTraceEnabled==false`. In that case I don't think we need the directory path?

Reply:
Fixed in 21c573b325a82b50473157cc43bda8ee4019363f